pydocx/util/memoize.py

```diff
@@ -5,7 +5,7 @@
     unicode_literals,
 )
 
-import collections
+from collections.abc import Hashable
```

Comment:
this was the reason for all this hassle. no longer exposed via collections in python3.10